app/lib/forms/question_types/base.rb

```diff
@@ -3,9 +3,10 @@ module QuestionTypes
     class Base
       attr_reader :name, :options
 
-      def initialize(name:, options: [])
+      def initialize(name:, options: [], opts: {})
         @name = name
         @options = options
```

Comment:
Would `opts = {}` be better than `opts: {}`? That way we aren't declaring `opts:..., options:...` and opts is purely an internal name.